platooning_strategic_IHP/src/platoon_strategic_ihp.cpp

```diff
@@ -1458,20 +2008,90 @@ namespace platoon_strategic_ihp
     {
     }
 
+    // UCLA: response callback for lead with operation
+    void PlatoonStrategicIHPPlugin::mob_resp_cb_leadwithoperation(const cav_msgs::MobilityResponse& msg)
+    { 
+    }
+
+    // UCLA: response callback for prepare to join (inherited from leader waiting)
+    void PlatoonStrategicIHPPlugin::mob_resp_cb_preparetojoin(const cav_msgs::MobilityResponse& msg)
+    {
+        /*
+            If leader notify the member to slow down and ACK the request,
+            start to check the gap and change lane when gap is large enough 
+        */
+
+        cav_msgs::PlanType plan_type = msg.plan_type;
+        bool isCreatingGap = (plan_type.type == cav_msgs::PlanType::PLATOON_CUTIN_JOIN);
```

Comment:
Extraneous parentheses.  Operator precedence makes them unnecessary.  This is typical of several places in this file.